src/params.rs

```diff
@@ -207,3 +212,58 @@ impl FromStr for GuestMemorySize {
 		Ok(memory_size)
 	}
 }
+
+/// Configure the kernels environment variables.
+#[derive(Debug, Clone, PartialEq)]
+pub enum EnvVars {
+	/// Pass all env vars of the host to the kernel.
+	Host,
+	/// Pass a certain set of env vars to the kernel.
+	Set(HashMap<String, String>),
+}
+impl Default for EnvVars {
+	fn default() -> Self {
+		Self::Set(HashMap::new())
+	}
+}
+impl<S: AsRef<str> + std::fmt::Debug + PartialEq<S> + From<&'static str>> From<&[S]> for EnvVars {
+	fn from(v: &[S]) -> Self {
+		if v.contains(&S::from("host")) {
+			return Self::Host;
+		}
```

Comment:
This means `-e K=v -e host` will silently drop `K`?

Reply:
Yes. I think a combination of the envs would be not really worth the effort, and I consider `-e host` a special case. But I can surely add a warning.

Reply:
(Added warning)